base_multi_company/models/multi_company_abstract.py

```diff
@@ -38,12 +39,19 @@ def _compute_no_company_ids(self):
                 record.no_company_ids = True
 
     @api.depends("company_ids")
-    @api.depends_context("company")
+    @api.depends_context("company", "_check_company_source")
```

Comment:
I think this part was missing to use the new variable name `_check_company_source_id`

Is not?

Reply:
Fixed in d0782cf5a12b63414d2167e26d1868e234ac1fd9